src/utils/defaultMap.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict-local
+
+/**
+ * A wrapper around Map, akin to Python's `defaultdict`.
```

Comment:
> src/utils/defaultMap.js

For the filename, I could go either way between `DefaultMap` and `defaultMap`. I didn't mean to set a precedent with `zulipVersion`, and sometimes I've thought `ZulipVersion` might be better. 😛 This is a small thing and not worth much fretting over though.

Reply:
Hmm, yeah, I didn't actually really think about that.

If I'm naming the file after the class, I think it's best for it to have the same name as the class. I'll make it `DefaultMap.js`.